src/main/java/com/aws/greengrass/provisioning/ProvisionContext.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.provisioning;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.Map;
+
+@AllArgsConstructor
+@Getter
+public class ProvisionContext {
+    String provisioningPolicy;
```

Comment:
This should be an enum

Reply:
As commented in the code, enum should exist in common library. Keeping it string here provides more flexibility when the enum is added later.